ftoolz/typing.py

```diff
@@ -25,3 +25,25 @@ def assert_some(a: Optional[A]) -> A:
     if a is None:
         raise TypingError('Item expected to be not None, None given.')
     return a
+
+
+def seq(it: Optional[Iterable[A]] = None) -> Seq[A]:
```

Comment:
I'm not sure about this function. Do we really need it? If so, then I would change the signature to `def seq(it: Iterable[A] = ()) -> Seq[A]:` to match functions like `list` or `set`.

Reply:
Well, it fixes an mypy issue. And it might be good as a definition of what a `Seq` really is - a tuple. ...But changing the param is definitely a good idea ;)